mapswipe_workers/mapswipe_workers/utils/tile_grouping_functions.py

```diff
@@ -271,12 +272,161 @@ def get_vertical_slice(slice_infos, zoom, width_threshold=40):
             }
 
             #####################
+            # TODO what does the line below exactly do?
             TileX = TileX + step_size
 
     logger.info("created vertical_slice")
     return raw_groups
 
 
+def remove_groups_within_other_groups(groups, zoom):
```

Comment:
@TahiraU remove this functions once you implemented the union for the project extents